pyiode/objects/equation.pyx

```diff
@@ -59,6 +59,9 @@ cdef class Equation:
     >>> variables.sample = "1960Y1:2015Y1"
     >>> eq_ACAF = Equation("ACAF", "(ACAF / VAF[-1]) := acaf1 + acaf2 * GOSF[-1] + acaf4 * (TIME=1995)")
     >>> eq_ACAF         # doctest: +NORMALIZE_WHITESPACE
+    # GDM> the __repr__ should be more Python-code like, ideally eval(repr_str) should be valid
+    #      (ie replace ":" by "=" and display "repr()" of comment, block and instruments, and MAYBE avoid showing
+    #       values which are the same as default values)
     Equation(lec: (ACAF / VAF[-1]) := acaf1 + acaf2 * GOSF[-1] + acaf4 * (TIME=1995),
```

Comment:
also I think it would be better if the name of the Equation was part of the `__repr__`

Reply:
OK. See PR #580